sdk/communication/communication-short-codes/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-11-09)
```

Comment:
Usually we leave the date as "(Unreleased)" and then have a separate PR to prepare for the release and set the actual date.

The CI pipeline will try a release if it finds a date, but it will only build without a release if its Unreleased.